imod_coupler/drivers/ribametamod/ribametamod.py

```diff
@@ -382,15 +388,17 @@ def update(self) -> None:
         if self.has_ribasim:
             self.exchange_rib2mod()
 
-        if self.has_ribasim and self.has_metaswap:
-            self.update_ribasim_metaswap()
-        elif self.has_ribasim and not self.has_metaswap:
-            self.update_ribasim()
-
         if self.has_ribasim:
-            # get realised values on basin boundary nodes and exchange correction flux
-            realised_basin_nodes = self.exchange.demand  # dummy value for now
-            self.exchange.to_modflow(realised_basin_nodes)
+            self.ribasim_infiltration_sum[:] = 0.0
```

Comment:
Can you move this to an exchange method?

Reply:
done, moved it to the exchange.demand method, so that immediately after setting the demands to ribasim, its integrated volumes are zeroed